Sources/Client/Client/Client+Channel.swift

```diff
@@ -43,20 +43,29 @@ public extension Client {
     ///   - type: a channel type.
     ///   - members: a list of members.
     ///   - extraData: a channel extra data.
+    ///   - namingStrategy: a naming strategy to generate a name and image for the channel based on members.
+    ///                     Only takes effect if `extraData` is `nil`.
     func channel(type: ChannelType = .messaging,
                  members: [User],
-                 extraData: ChannelExtraDataCodable? = nil) -> Channel {
-        Channel(type: type,
-                id: "",
-                members: members,
-                invitedMembers: [],
-                extraData: extraData,
-                created: .init(),
-                deleted: nil,
-                createdBy: nil,
-                lastMessageDate: nil,
-                frozen: false,
-                config: .init())
+                 extraData: ChannelExtraDataCodable? = nil,
+                 namingStrategy: ChannelNamingStrategy? = Channel.DefaultNamingStrategy(maxUserNames: 1)) -> Channel {
```

Comment:
I thought default max user names would be 3

Reply:
Yes, I thought 3 will be great, then I saw an example from @VojtaStavik and then I think 1 is good for iPhone and 3 for iPad.

Reply:
For iPhone, it's not enough space for 3.

Reply:
OK, just wanted to point out. 1 is good